<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(204, 213, 174);
selection-background-color: rgb(169, 176, 143);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>1021</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView
{
	background-color: rgb(169, 176, 143);
}

QHeaderView::section
{
	background-color: rgb(169, 176, 143);
	font: 8pt &quot;Century Gothic&quot;;
}

QTableWidget::item::selected 
{
 
	color: rgb(254, 253, 239);
	background-color: rgb(105, 109, 88);
	font: 8pt &quot;Century Gothic&quot;;
}
 </string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(234, 241, 210);</string>
    </property>
    <item>
     <property name="text">
      <string>Все</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Тип мероприятия:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="placeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(234, 241, 210);</string>
    </property>
    <item>
     <property name="text">
      <string>Все</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Место проведения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата проведения:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(105, 109, 88);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время проведения:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 241, 210);
font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 241, 210);
font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Узнать подробнее:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moreButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(105, 109, 88);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Подробнее</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="more">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(234, 241, 210);
font: 9pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Введите ID</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="anydate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
	border: 1px solid;
    background: rgb(255, 255, 255);
}
QCheckBox::indicator:checked {
    background: rgb(125, 130, 105);
}
QCheckBox {
	font: 9pt &quot;Century Gothic&quot;;
}</string>
    </property>
    <property name="text">
     <string>любая дата</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="anytime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
	border: 1px solid;
    background: rgb(255, 255, 255);
}
QCheckBox::indicator:checked {
    background: rgb(125, 130, 105);
}
QCheckBox {
	font: 9pt &quot;Century Gothic&quot;;
}</string>
    </property>
    <property name="text">
     <string>любое время</string>
    </property>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>picture</zorder>
   <zorder>tableWidget</zorder>
   <zorder>typeBox</zorder>
   <zorder>label</zorder>
   <zorder>placeBox</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>searchButton</zorder>
   <zorder>label_4</zorder>
   <zorder>dateEdit</zorder>
   <zorder>timeEdit</zorder>
   <zorder>label_5</zorder>
   <zorder>moreButton</zorder>
   <zorder>more</zorder>
   <zorder>anydate</zorder>
   <zorder>anytime</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1323</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
